docs/about/release_notes.rst

```diff
@@ -27,6 +27,134 @@ Release Notes
  Miscellaneous:
    -
 
+
+Version 8.0
+-----------
+
+In this version of ERT we have cleaned up a lot of unused analysis keywords. Some have been deprecated,
+while others that were previously deprecated have now been removed. If they are present in your
+configuration, you will get a message in the suggestor at startup:
+
+.. image:: version-8.0-suggestor.png
+
+We encourage everyone to keep their configuration updated and remove and replace deprecated keywords.
+
+Another big change this release is the change to a streaming algorithm, which updates a
```

Comment:
```suggestion
This release introduces the streaming algorithm, which updates a
```
The `in` was missing, but I rewrote to get rid of double use of the word `change`